packages/gafl-webapp-service/src/processors/__tests__/licence-type-display.spec.js

```diff
@@ -107,3 +110,15 @@ describe('isPhysical', () => {
     expect(result).toEqual(undefined)
   })
 })
+
+describe('recurringLicenceTypeDisplay', () => {
+  it.each([
+    ['Salmon and sea trout', null, ' salmon and sea trout'],
+    ['Trout and coarse', '2', ' trout and coarse (2 rod)'],
+    ['Trout and coarse', '3', ' trout and coarse (3 rod)']
+  ])('returns correct licence type', (licenceType, numberOfRods, expected) => {
```

Comment:
identical description for three tests - please differentiate them from one another by using the parameters in the test description